spec/models/item_spec.rb

```diff
@@ -57,6 +57,29 @@
       expect(Item.by_size("4").length).to eq(2)
     end
 
+    it "->housing_a_kit returns all items which belongs_to (house) a kit" do
```

Comment:
In general, if the first word of the test description isn't "should", you should use the `specify` keyword instead of `it`. They are aliases of each other - it's just a readability thing.

Reply:
changed in #4665 [7d25b3a](https://github.com/rubyforgood/human-essentials/pull/4665/commits/7d25b3aeb4ee4f4b55e7d58d374c39b7f77cbb1a)